src/resources/extensions/quarto/video/video.lua

```diff
@@ -180,6 +182,29 @@ local helpers = {
   ["getSnippetFromBuilders"] = getSnippetFromBuilders
 }
 
+local function asciidocVideo(src, height, width, title, start, _aspectRatio)
+
+  -- makes an asciidoc video raw block
+  -- see https://docs.asciidoctor.org/asciidoc/latest/macros/audio-and-video/
+  local makeAdoc = function(id, type) 
+    return pandoc.RawBlock("asciidoc", 'video::' .. id .. '[' .. type .. ']\n\n')
```

Comment:
As the RawBlock is text, you may want to consider extracting this out to the helper and adding a unit test.  

It also seems a bit strange that the source is being called `id` in `makeAdoc` maybe rename to `src`.